apps/accounts/views.py

```diff
@@ -65,6 +65,9 @@ def inner(self, request):
         data = request.GET if request.method == 'GET' else request.DATA
         if 'code' not in data:
             return Response({'error': 'No code provided.'}, status=422)
+        elif ('fxa_state' not in request.session or
+                request.session['fxa_state'] != data['state']):
```

Comment:
I had only included the user ID so that I knew when to invalidate it. That should likely be handled by django already though so maybe I can pull it out.